### PR TITLE
added knex connection to app.js

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -2,11 +2,25 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const path = require('path')
 const morgan = require('morgan');
+const knex = require('knex');
+const knex2 = interopRequireDefault(knex);
+const objection = require('objection');
+const knexfile = require('./knexfile');
+
+function interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```
Comment (kkarimi) [inline backend/app.js]: why is this necessary? can't see it in the example from their repo

Comment (sentayhuJo) [reply backend/app.js]: if its work and he is able to connect the front end, do it. but we will come up with the new API end point soon.

File: backend/knexfile.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
+
```
[the remark this answers is not in the document]
Comment (DwinaTech) [reply backend/knexfile.js]: Done

File: backend/app.js [hunk 3]
```diff
@@ -2,11 +2,25 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const path = require('path')
 const morgan = require('morgan');
+const knex = require('knex');
+const knex2 = interopRequireDefault(knex);
```
[the remark this answers is not in the document]
Comment (DwinaTech) [reply backend/app.js]: Removed and used the connection as explained in objection example

File: backend/app.js (hunk 1 above)
Comment (DwinaTech) [reply backend/app.js]: Removed and used the connection as explained in objection example

File: backend/app.js [hunk 4]
```diff
@@ -1,29 +1,37 @@
...
 const morgan = require('morgan');
-
+const { Model } = require('objection');
+const knexfile = require('./knexfile');
 const router = require('./routes/api');
 
 const app = express();
 
+// Initialize knex.
+const knexConfig = (knexfile.development);
```
Comment (kkarimi) [inline backend/app.js]: Fine for now, but in future this will be coming from an environment variable
